generators/app/templates/src/app/components/Checkbox/layout.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Checkbox({
+  className,
+  inputClassName,
+  labelClassName,
+  label,
+  name,
+  isChecked,
+  onToggle,
+  disabled,
+  required
+}) {
+  return (
+    <div className={className}>
```

Comment:
`containerClassName` ?

Reply:
I see the point, but as we are used to having a `className` prop for almost every component, someone may be tempted to use it without any result.